android/source/VideoLocker/src/org/edx/mobile/module/analytics/SegmentTracker.java

```diff
@@ -13,16 +13,18 @@
 import com.segment.analytics.Traits;
 
 public class SegmentTracker {
-    
+
+    /* Singleton instance of Analytics */
     private Analytics analytics;
     
     public SegmentTracker(Context context) {
         try {
             String writeKey = Environment.getInstance().getConfig().getSegmentIOWriteKey();
-            String debugging = context.getString(R.string.analytics_debug);
-            int queueSize = context.getResources().getInteger(R.integer.analytics_queue_size);
 
-            if(writeKey!=null) {
+            if (writeKey != null) {
```

Comment:
Can we add a Log info message here if `writeKey` is null?  I want to be able to flag the situation when analytics events will not be captured.  Also, this should never happen in edX's production configuration.